sdk/identity/Azure.Identity/src/DefaultAzureCredentialOptions.cs

```diff
@@ -70,5 +84,16 @@ public class DefaultAzureCredentialOptions : TokenCredentialOptions
         /// Specifies whether the <see cref="AzureCliCredential"/> will be excluded from the <see cref="DefaultAzureCredential"/> authentication flow.
         /// </summary>
         public bool ExcludeAzureCliCredential { get; set; } = false;
+
+        /// <summary>
+        /// Specifies whether the <see cref="VisualStudioCredential"/> will be excluded from the <see cref="DefaultAzureCredential"/> authentication flow.
+        /// </summary>
+        public bool ExcludeVisualStudioCredential { get; set; } = false;
```

Comment:
Do we include developer credentials by default?

Reply:
For now I was. I'm assuming this might change once we have fully defined the intended developer credential workflow. cc: @jonathandturner

Reply:
Is it allowed to make that change for defaults in the next preview?

Reply:
Yes we don't guarantee backwards compatibility between preview releases.